collector.go

```diff
@@ -228,6 +228,8 @@ func pduValueAsString(pdu *gosnmp.SnmpPDU, typ string) string {
 		return strconv.FormatUint(uint64(pdu.Value.(uint)), 10)
 	case int64:
```

Comment:
Ok, so it's an SNMP spec issue, not a library issue.  I misunderstood your IRC comment as implying it was an issue with the library. 👍